decorator.py

```diff
@@ -0,0 +1,35 @@
+"""
+This task contains a few parts:
+Write a custom function which returns your name or username.
+Write a decorator which allows access for your user and prints 
+�Permission denied� for other users.
+
+Tips:
+Create get_user_info function which returns your name or username.
+Create decorator which raises Exception if get_user_info returns not 
+your name or username
+Create check_perms function decorated by decorator. This function
+returns Allowed access if user validation is successful.
+"""
+
+
+def my_is_name():
+    return 'sergiy'
+
+
+def decorator(my_name):
```

Comment:
I suggest using the name like `allowed_users`, etc.

Reply:
done